pkg/api/admin/api/config.go

```diff
@@ -3,7 +3,7 @@ package api
 import (
 	"crypto/x509"
 
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```

Comment:
Any particular reason for this change here?

Reply:
gofmt changed it, wth